src/components/ScanBoxDetails.js

```diff
@@ -63,40 +65,46 @@ const ScanBoxDetails = ({ ...props }) => (<div className="scanbox-details">
         />
       </Col>
     </Row>
+    {props.matrix.q5.answer &&
```

Comment:
after long consideration I think it is better to use totalETH from stats.money, not q5, please change it. like with tokens